Source/Source/Harmony/JobDriver_Patch.cs

```diff
@@ -12,10 +18,20 @@ public class JobDriver_Patch
         [HarmonyPatch(typeof(JobDriver), nameof(JobDriver.DriverTick))]
         public class DriverTick
         {
-            [HarmonyPrefix]
-            public static void Prefix(Pawn ___pawn)
+            private static FieldInfo pawnField = AccessTools.Field(typeof(JobDriver), nameof(JobDriver.pawn));
+            private static FieldInfo ourTransplant = AccessTools.Field(typeof(ForbidUtility_Patch), nameof(ForbidUtility_Patch.currentToilWorker));
+
+            [HarmonyTranspiler]
```

Comment:
I'm not really a big fan of turning everything into transpilers. I find them impossible to read. I won't ever be able to fix it when something breaks here.